presto-docs/src/main/sphinx/release/release-0.279.rst

```diff
@@ -0,0 +1,64 @@
+=============
+Release 0.279
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix bug in CombineApproxPercentileFunctions optimization. It fix the bug where the CombineApproxPercentileFunctions optimization rule fails when there are duplicate aggregation expressions to be optimized.
+* Fix disagg coordinator task limit enforcement.
+* Fix for weighted fair scheduling in disaggregated coordinator.
+* Fix max(x, n) and min(x, n) returning NULL on window aggregations.
+* Fix weighted scheduling for disaggregated coordinator.
+* Add :func:`array_sort_desc` function to sort an array in the descending order.
+* Add :func:`map_remove_null_values` to remove all the entries where the value is null from the given map.
+* Add :func:`map_top_n_values` to return top N values of the provided map. An optional lambda comparator can also be passed as parameter for custom sorting of the values.
+* Add :func:`map_top_n` to truncates map items. Keeps only the top N elements by value.
+* Add :func:`remove_nulls` function to remove null elements from an array.
+* Add ``CONTROL`` as a new ``QueryType``.  The CONTROL queryType represents statements of session control and transaction control types.
+* Add an optimization which removes redundant distinct if the output is already distinct after a group by operation. The optimization is controlled by session property `remove_redundant_distinct_aggregation` which is default to false.
+* Add functions :func:`array_min_by`, :func:`array_max_by`, to find the smallest or largest element of an array when applying a custom measuring function.
+* Add property ``spark_executor_allocation_strategy_enabled`` to auto-tune spark max executor count (``spark.dynamicAllocation.maxExecutors``)  based on input data. Only required if ``spark_resource_allocation_strategy_enabled`` is not already enabled.
+* Add property ``spark_hash_partition_count_allocation_strategy_enabled`` to auto-tune hash partition count (``hash_partition_count``) based on input data. Only required if ``spark_resource_allocation_strategy_enabled`` is not already enabled.
```

Comment:
Line 26 and 27 should be highlighted under "Spark Changes"

eg. https://prestodb.io/docs/current/release/release-0.276.html#spark-changes